indexer/src/main/java/au/org/aodn/esindexer/utils/MapperUtils.java

```diff
@@ -31,10 +31,11 @@ public static class Contacts {
         protected LinkedHashSet<LinkModel> onlineResources = new LinkedHashSet<>();
     }
 
-    public static String mapContactsRole(CIResponsibilityType2 ciResponsibility) {
+    public static List<String> mapContactsRole(CIResponsibilityType2 ciResponsibility) {
         CodeListValueType roleCode = ciResponsibility.getRole().getCIRoleCode();
-        return roleCode != null ?
-                roleCode.getCodeListValue() : "";
+        return Collections.singletonList(
+                roleCode != null ?
+                roleCode.getCodeListValue() : "");
```

Comment:
null check for `roleCode.getCodeListValue()` too, `roleCode != null && roleCode.getCodeListValue() != null`